sdk/identity/azure-identity/azure/identity/_credentials/chained.py

```diff
@@ -6,11 +6,12 @@
 from typing import Any, Optional, TYPE_CHECKING
 from azure.core.exceptions import ClientAuthenticationError
 
+from azure.core.credentials import AccessToken
 from .. import CredentialUnavailableError
 from .._internal import within_credential_chain
 
 if TYPE_CHECKING:
-    from azure.core.credentials import AccessToken, TokenCredential
+    from azure.core.credentials import TokenCredential
```

Comment:
nit: I don't think we need to put this under TYPE_CHECKING anymore (IIRC it was fixed/importable since azure-core >=1.25)

Reply:
You are correct.

But currently identity depends on azure-core 1.11.

I am not sure if it is worth bumping the core dependency version for the sake of typing purpose.